sensor_msgs/PointCloud2 pointcloud  
uint8[] connected_labels
uint8 connected_labels_n
uint8[] color_labels
uint8 color_labels_n